src/funding.rs

```diff
@@ -410,51 +410,47 @@ impl AnsibleProvisioner {
         let default_token_amount = U256::from_str(DEFAULT_TOKEN_AMOUNT).unwrap();
         let default_gas_amount = U256::from_str(DEFAULT_GAS_AMOUNT).unwrap();
 
-        let tokens_for_each_uploader = options.token_amount.unwrap_or(default_token_amount);
-        let gas_for_each_uploader = options.gas_amount.unwrap_or(default_gas_amount);
+        let token_amount = options.token_amount.unwrap_or(default_token_amount);
+        let gas_amount = options.gas_amount.unwrap_or(default_gas_amount);
 
-        println!("Transferring {tokens_for_each_uploader} tokens and {gas_for_each_uploader} gas tokens to each uploader");
-        debug!("Transferring {tokens_for_each_uploader} tokens and {gas_for_each_uploader} gas tokens to each uploader");
+        println!("Transferring {token_amount} tokens and {gas_amount} gas tokens to each uploader");
+        debug!("Transferring {token_amount} tokens and {gas_amount} gas tokens to each uploader");
 
-        for (vm, sks_per_machine) in all_secret_keys.iter() {
-            debug!("Transferring funds for uploader vm: {}", vm.name);
-            for sk in sks_per_machine.iter() {
+        for (vm, vm_secret_keys) in all_secret_keys.iter() {
+            println!("Transferring funds for uploader vm: {}", vm.name);
+            for sk in vm_secret_keys.iter() {
                 sk.address();
 
-                if !tokens_for_each_uploader.is_zero() {
-                    debug!(
-                        "Transferring {tokens_for_each_uploader} tokens for uploader vm: {} with public key: {}",
-                        vm.name,
-                        sk.address()
-                    );
```

Comment:
Could we retain the debug lines everywhere? It contains details about the `vm`, which will be useful to know.

Reply:
The VM is printed just above, so it's obvious what one is currently being processed.